skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/services/BrevredigeringService.kt

```diff
@@ -454,7 +455,7 @@ class BrevredigeringService(
                         pesysData = pesysData.brevdata,
                         saksbehandlerValg = saksbehandlerValg,
                     ),
-                    felles = pesysData.felles.copy(signerendeSaksbehandlere = SignerendeSaksbehandlere(signaturSignerende, signaturAttestant))
+                    felles = pesysData.felles.kopier(signerendeSaksbehandlere = SignerendeSaksbehandlereImpl(signaturSignerende, signaturAttestant))
```

Comment:
Siden vi allerede har optin for dataklassene her så burde vi vel kunne bruke `copy` metoden fra FellesImpl?

Reply:
`felles` har type `Felles` her, altså interfacet, så i så fall må vi caste over til `FellesImpl`, eller at data-klassa i `BrevdataResponse` er FellesImpl da.